datarepo-clienttests/src/main/resources/configs/KubeConfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "KubeConfig",
+  "description": "One user ingests many files into one dataset while we manipulate the kubernetes pods",
+  "serverSpecificationFile": "shdev.json",
```

Comment:
I think it was a good idea to change the other configs from mmdev -> localhost so other people aren't accidentally hitting our personal namespaces when they run the tests. Perhaps change this one to localhost too?